13-tool-usage-cycle-tracking/README.md

```diff
@@ -257,28 +281,71 @@ Each power monitor is given an instance number from 1-4. The instance numbers co
 
-* `power_activity_shutdown_secs_[1-4]`: the duration in seconds for how long it takes for the power level to stabilize when the equipment is stopped. Alerts are suppressed for this period when a line goes from active to inactive. The default value is `0`.
+ * `alert_vibration_activity_line` 1-4: indicates which activity line shows the activity of the tool. When this environment variable is defined, it correlates vibration alerts based on the activity of the tool and the vibration sensed: 
+
+  * when the tool is inactive, a vibration alert is sent when the vibration is not `none`.
+  * when the tool is active, a vibration alert is sent when the vibration is not `normal`.
+
+The alert is triggered on the instance specified in `alert_vibration_activity_line`.  When `alert_vibration_activity_line` is `0` or undefined, the vibration data and alerts are present on all lines.
+
+Vibration alerts are suppressed for the startup and shutdown period given for the corresponding line.
+
+
+### Tool Startup and Shutdown Duration
+
+These variables describe how long it takes for the tool to startup or shutdown, or equivalently, for the electrical load and vibration to reach steady state. Power and vibration alerts related to activity are suppressed during the startup and shutdown period to avoid false alerts.
+
+* `power_activity_startup_secs_[1-4]`: the duration in seconds for how long it takes the power level to stabilize on becoming active. Alerts are suppressed for this period when a line goes from inactive to active. The default value is `0`.
+
+* `power_activity_shutdown_secs_[1-4]`: the duration in seconds for how long it takes for the power level to stabilize on becoming inactive.  Alerts are suppressed for this period when a line goes from active to inactive. The default value is `0`.
+
+
+### Determining Vibration Thresholds
+
+The raw vibration measurement is used to determine the vibration level is derived from the movements of the accelerometer. The value is somewhat arbitrary and not calibrated, although it is consistent.
+
+The simplest way to find out the values for the vibration variables is to measure the vibration reported when the tool is off and during normal use. Here's an outline of the process:
+
+1. Start with none of the vibration environment variables defined, and the tool inactive.
+2. Set `heartbeat_mins` to `1` to enable more frequent monitoring.
+3. Enable vibration sensing by setting `alert_under_vibration` to `1`.
+4. With the tool still inactive, inspect the events in Notehub sent to `power.qo`, in particular the `vibration_raw` value.
+5. Make a note of the highest reported `vibration_raw` value. This will serve as a guide for setting the `alert_off_vibration` variable.
+6. Switch on the tool, and when fully started, continue monitoring the events for a number of minutes, using the tool in it's normal modes of operation.
```

Comment:
> it's

its

Reply:
It's funny how I know this, but still write it incorrectly.